python/pyspark/sql/conf.py

```diff
@@ -66,11 +59,7 @@ def get(
 
     @since(2.0)
     def unset(self, key: str) -> None:
-        """Resets the configuration property for the given key.
-
-        .. versionchanged:: 3.4.0
-            Supports Spark Connect.
-        """
+        """Resets the configuration property for the given key."""
```

Comment:
This comment is not related to the current PR. However, I have a question about the accuracy of this document. Unsets?

Reply:
Lol let's fix it since we're here.

Reply:
Actually, I think it's fine. It will reset it to its default value .. in a way :-).

Reply:
Scala side has the same doc so let's leave it here for now.

Reply:
Yes, I am okay with it.